charts/dave/charts/admin-portal/values.yaml

```
@@ -32,13 +32,13 @@ podSecurityContext:
   # fsGroup: 2000
 
 securityContext:
-  {}
-  # capabilities:
-  #   drop:
-  #   - ALL
-  # readOnlyRootFilesystem: true
-  # runAsNonRoot: true
-  # runAsUser: 1000
+#  {}
```

YAMLlint (1.37.1)

[warning] comment not indented like content

(comments-indentation)